src/Symfony/Component/Security/Http/Authenticator/AccessToken/BodyAccessTokenExtractor.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Http\Authenticator\AccessToken;
+
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * Extracts a token from the body request.
+ *
+ * @author Florent Morselli <florent.morselli@spomky-labs.com>
+ */
+final class BodyAccessTokenExtractor implements AccessTokenExtractorInterface
```

Comment:
Is this part of any common spec? If yes we should reference it in the doc block, if no, maybe should we not introduce this extractor in the core framework.

Reply:
+1 for `FormEncodedBodyExtractor`

Reply:
OK done in the last commit